doc/classes/Transform3D.xml

```diff
@@ -159,111 +164,118 @@
 				[b]Note:[/b] Due to floating-point precision errors, consider using [method is_equal_approx] instead, which is more reliable.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="AABB" />
 			<param index="0" name="right" type="AABB" />
 			<description>
-				Transforms (multiplies) the [AABB] by the given [Transform3D] matrix.
+				Transforms (multiplies) the [AABB] by this transformation matrix.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="PackedVector3Array" />
 			<param index="0" name="right" type="PackedVector3Array" />
 			<description>
-				Transforms (multiplies) each element of the [Vector3] array by the given [Transform3D] matrix.
+				Transforms (multiplies) every [Vector3] element of the given [PackedVector3Array] by this transformation matrix.
+				On larger arrays, this operation is much faster than transforming each [Vector3] individually.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Plane" />
 			<param index="0" name="right" type="Plane" />
 			<description>
-				Transforms (multiplies) the [Plane] by the given [Transform3D] transformation matrix.
+				Transforms (multiplies) the [Plane] by this transformation matrix.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Transform3D" />
 			<param index="0" name="right" type="Transform3D" />
 			<description>
-				Composes these two transformation matrices by multiplying them together. This has the effect of transforming the second transform (the child) by the first transform (the parent).
+				Transforms (multiplies) this transform by the [param right] transform.
+				This is the operation performed between parent and child [Node3D]s.
+				[b]Note:[/b] If you need to only modify one attribute of this transform, consider using one of the following methods, instead:
+				- For translation, see [method translated] or [method translated_local].
+				- For rotation, see [method rotated] or [method rotated_local].
+				- For scale, see [method scaled] or [method scaled_local].
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Vector3" />
 			<param index="0" name="right" type="Vector3" />
 			<description>
-				Transforms (multiplies) the [Vector3] by the given [Transform3D] matrix.
+				Transforms (multiplies) the [Vector3] by this transformation matrix.
 			</description>
 		</operator>
 		<operator name="operator *">
 			<return type="Transform3D" />
 			<param index="0" name="right" type="float" />
 			<description>
-				This operator multiplies all components of the [Transform3D], including the [member origin] vector, which scales it uniformly.
+				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, also resizing the [member basis].
```

Comment:
```suggestion
				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, scaling the [member basis].
```
"Resize" isn't really applicable

Reply:
Something like "Also" feels necessary though. The `origin` is affected, but _also_ the `basis`.

Reply:
But this sentence says "the scale (of the basis) is changed, as is the basis", the origin isn't part of the scale